userspace/libsinsp/test/events_param.ut.cpp

```diff
@@ -160,10 +160,11 @@ TEST_F(sinsp_with_test_input, sockaddr_empty_param) {
 	evt = add_event_advance_ts(increasing_ts(),
 	                           1,
 	                           PPME_SOCKET_CONNECT_X,
-	                           3,
+	                           4,
 	                           fd,
```

Comment:
Isn't this param supposed to be the return value?

Reply:
Actually, I noticed I had to rebase it, so I'm gonna fix it.